lib/beaker-answers/pe_conf.rb

```diff
@@ -57,7 +56,7 @@ def configuration_hash
         _generate_1_0_data
       when '2.0'
         _generate_2_0_data
-      else raise(RuntimeError, "Unknown how to produce pe.conf data for meep_schema_version: '#{meep_schema_version}'")
+      else raise "Unknown how to produce pe.conf data for meep_schema_version: '#{meep_schema_version}'"
```

Comment:
It's probably true that just calling raise will produce a RuntimeError, but I don't think we should remove the explicit Error, at least then we know someone thought about it.
So let's drop this one.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_style/#styleredundantexception disabled